src/index.ts

```diff
@@ -4,22 +4,24 @@ import { Client as DiscordClient } from "discord.js";
 import "reflect-metadata";
 import { CONFIG } from "./config";
 import { OnGuildMemberAdd, OnMessage } from "./events";
-import "./injects";
```

Comment:
I'm sus of this import delete >.> I would guess the commands don't work with this gone >.>

Reply:
woops - does it still work? I didn't mean to delete this line!